application/comit_node/src/http_api/mod.rs

```diff
@@ -398,4 +407,15 @@ mod tests {
             r#""ad2652ca-ecf2-4cc6-b35c-b4351ac28a34""#
         )
     }
+
+    #[test]
+    fn http_peer_id_serializes_correctly_to_json() {
+        let peer_id = PeerId::random();
+        let want = format!("\"{}\"", peer_id);
```

Comment:
Hum, I don't think is a great test because if `serde_json::to_string` uses the `Display` trait underneath then there may a bug and you would not see it.

Can you initialize a `PeerId` from a `String` for example? I'd like to see that the test is actually done against the string slice `"QmfUfpC2frwFvcDzpspnfZitHt5wct6n4kpG5jzgRdsxkY"`

Reply:
Implemented as suggested.